Gemfile

```diff
@@ -39,7 +39,7 @@ group :development do
   gem 'rack-mini-profiler', '~> 2.0'
   gem 'listen', '~> 3.3'
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
-  gem 'spring', '~> 3.1'
+  # gem 'spring', '~> 3.1'
```

Comment:
i would be slightly inclined to just remove this line and the comment above, but wouldn't hold up merging over it